liwords-ui/src/settings/avatar_cropper.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useCallback, useEffect } from 'react';
+import { Button, Modal, Slider } from 'antd';
+import Cropper from 'react-easy-crop';
+import { useMountedState } from '../utils/mounted';
+import { Area } from 'react-easy-crop/types';
+
+type Props = {
+  file?: Blob;
+  onCancel: () => void;
+  onSave: (imageDataUrl: string) => void;
+  onError: (errorMessage: string) => void;
+};
+
+export const AvatarCropper = React.memo((props: Props) => {
+  const { useState } = useMountedState();
+  const [dataUrl, setDataUrl] = useState<string | undefined>(undefined);
+  const [crop, setCrop] = useState({ x: 0, y: 0 });
+  const [zoom, setZoom] = useState<number | undefined>(1);
+  const [croppedArea, setCroppedArea] = useState<undefined | Area>(undefined);
+  const { file, onSave } = props;
+  const handleOnSave = useCallback(() => {
+    if (!croppedArea || !dataUrl) {
+      return;
+    }
+    let image = new Image();
+    image.onload = () => {
+      const canvas = document.createElement('canvas'),
+        ctx = canvas.getContext('2d');
+      canvas.width = 96;
+      canvas.height = 96;
+      ctx!.drawImage(
+        image,
+        croppedArea.x,
+        croppedArea.y,
+        croppedArea.width,
+        croppedArea.height,
+        0,
+        0,
+        canvas.width,
+        canvas.height
+      );
+      const newDataUrl = canvas.toDataURL('/image.jpeg', 1);
```

Comment:
`'/image.jpeg'` in two places (previous code used `'image/jpeg'`)

Reply:
Fixed! Thanks